crd/src/lib.rs

```diff
@@ -27,11 +29,35 @@ impl Crd for KafkaCluster {
     const CRD_DEFINITION: &'static str = include_str!("../kafkaclusters.crd.yaml");
 }
 
-#[allow(non_camel_case_types)]
 #[derive(Clone, Debug, Deserialize, JsonSchema, Serialize)]
-pub enum KafkaVersion {
-    #[serde(rename = "2.6.0")]
-    v2_6_0,
+pub struct KafkaVersion {
+    kafka_version: String,
+    scala_version: Option<String>,
+}
+
+impl KafkaVersion {
+    pub const DEFAULT_SCALA_VERSION: &'static str = "2.13";
+
+    pub fn get_kafka_version(&self) -> &str {
```

Comment:
I opted for getters in this case because I wanted the functionality to provide a default scala version in that getter instead of providing direct access to that field.